packages/core/src/components/selectable/selectable.tsx

```diff
@@ -0,0 +1,58 @@
+import { h, Component, Host, Prop, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'dso-selectable',
+  styleUrl: 'selectable.scss',
+  shadow: true
+})
+export class Selectable {
+  @Prop()
+  type!: 'checkbox' | 'radio';
+
+  @Prop()
+  identifier!: string;
+
+  @Prop()
+  name?: string;
+
+  @Prop()
+  value!: string;
+
+  @Prop()
+  invalid?: boolean;
+
+  @Prop()
+  disabled?: boolean;
+
+  @Prop()
+  required?: boolean;
+
+  @Prop()
+  checked?: boolean;
+
+  @Event({
+    eventName: 'dsoChange'
+  })
+  change!: EventEmitter<any>;
+
+  render() {
+    return (
+      <Host>
+        <input
+          type={this.type}
+          id={this.identifier}
+          value={this.value}
+          name={this.name}
+          aria-invalid={this.invalid ? 'true': 'false'}
```

Comment:
Kan dit niet met `aria-invalid={this.invalid}`? Of ziet hij hem dan als `undefined`, en niet als `false`?

Reply:
Bij falsy values verdwijnt het argument.. Ik weet alleen niet of _geen_ aria-invalid attribuut gelijk is aan `aria-invalid="false"`..

@timveld Kan jij hier wat over roepen?